collect-settings.rst

```diff
@@ -175,13 +175,182 @@ Admin settings allow you to :ref:`restrict which General Settings are seen by us
 Import/Export settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:guilabel:`Import/Export Settings` lets you configure other devices identical to your current device simply by scanning the QR code. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When scanned by the ODK Collect app with another device, these settings are imported.
+When you click on the :guilabel:`Import/Export Settings`, you see a QR Code and a few options. 
 
-You can also import settings from a QR code saved to the device, by selecting :guilabel:`Select Code from SD Card`
+.. image:: /img/collect-settings/import-settings.*
+  :alt: Import/export settings menu of ODK Collect
+  :class: device-screen-vertical
 
-.. warning:: 
+QR Code
+""""""""
+
+QR Code or the Quick Response Code is a two dimensional barcode. QR codes can be used to configure Collect on many devices. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When this QR code is scanned from the ODK Collect app on another device, these settings are imported.
+
+Sharing QR code
+""""""""""""""""
+
+You can click on the |share| icon to share the generated barcode as an image. When you click on it, it displays a list of applications like *whatsapp*, *facebook* , *hangouts* to name a few, which can be used to share the generated QR code. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
```

Comment:
So I should share the QR code via Facebook?

Reply:
You have an option to share it with facebook messenger provided you have it installed on your phone.

Reply:
Yes, but should I? Is that an effective way of sharing QR codes? Can I share it directly to other phones without going through an app like Facebook?

Reply:
No sir, that's not effective since it contains data like the admin password. Apologies for listing the wrong items.

Reply:
That was my point. You said it lists several items... but then what is the user supposed to do with that information? Is there one best one? Are none of them a good idea?

Reply:
I put it in the form of a warning sir.